src/main/java/com/example/board/domain/common/entity/BaseEntity.java

```diff
@@ -0,0 +1,23 @@
+package com.example.board.domain.common.entity;
+
+import javax.persistence.Column;
+import javax.persistence.EntityListeners;
+import javax.persistence.MappedSuperclass;
+import lombok.Getter;
+import org.springframework.data.annotation.CreatedBy;
+import org.springframework.data.annotation.LastModifiedBy;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+@Getter
+@MappedSuperclass
+@EntityListeners({AuditingEntityListener.class})
+public class BaseEntity extends BaseTimeEntity{
```

Comment:
```suggestion
public abstract class BaseEntity extends BaseTimeEntity{
```
불완전한객체를 생성할 수 없도록 해보죠!

Reply:
오 추상클래스는 객체화를 못하죠. 좋습니다!